### DISCUSSION TITLE
Official Vector logo (SVG) #branding

### DISCUSSION THREAD

**Discussion by karol-depka** [pmndrs/zustand#1623] [General]
![zustand--logo512 png--vectorizer ai](https://user-images.githubusercontent.com/958486/218346783-72be5ae3-b953-4dd7-b239-788a882fdad6.svg)

Attaching idea from favicon via vectorizer.ai . Needs some cleanup probably.

**Comment by karol-depka** [comment]
Ideally, this would get to svgporn.com at some point.

**Comment by lumpinif** [comment]
really having trouble to compress it down...it is so big even can block the UI

**Comment by cicero-mello** [comment]
Nice!

Based on this, I made a more minimalist version (1 color). 
If someone wants:
![zustand](https://github.com/user-attachments/assets/c9651a03-dad6-440a-8628-4c3f643f8e3c)
